packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts

```diff
@@ -38,20 +38,22 @@ const meta: Meta<typeof OnyxSelect> = {
       /**
        * Decorator that simulates the load more functionality so we can show it in the stories.
        */
-      (story, ctx) => ({
-        components: { story },
-        setup: () => {
-          const { isLazyLoading, handleLoadMore, options } = useLazyLoading(ctx.args.options);
+      (story, ctx) => {
```

Comment:
The search example does no longer work

Reply:
Works for me, can you extend on this? What is the issue?

Reply:
https://github.com/SchwarzIT/onyx/assets/67898185/9a6a943f-2e5c-4e8e-af3b-0d53d5a4de59

Also when re-opening the select, the focus is not set on the search inut

Reply:
Cannot reproduce this 🤔 Maybe it's because I forgot to push?

Reply:
The other issue is an existing issue

Reply:
I still have the issue but only when using it on the "Docs" page. The single Story works fine